package.json

```diff
@@ -38,6 +38,7 @@
     "select2": "^4.0.13",
     "shortcut-buttons-flatpickr": "^0.4.0",
     "stimulus": "^3.0.1",
+    "tom-select": "^2.0.0",
```

Comment:
Confess, you chose just on the name?

Reply:
![](https://media.giphy.com/media/Hf2hyLE6l4Myk/giphy.gif)

Reply:
I'll write some more notes when this is ready for review, but basically the selection criteria was:
- small and simple package
- fully featured and customisable
- doesn't depend on jQuery
- uses modern Javascript
- plays nicely with StimulusJS
- plays nicely with other things like Turbo and StimulusReflex

That leaves two options, which were SlimSelect and TomSelect, and the consensus from other devs I've asked is that TomSelect is slightly better. I also gave them both a quick try and this one definitely seemed easier to use and generally nicer.